packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -79,7 +79,7 @@ export const enableModernEventSystem = true;
 export const enableNewReconciler = __VARIANT__;
 
 // TODO: This does not currently exist in the Lanes implementation.
-export const enableDebugTracing = false;
+export const enableDebugTracing = !__VARIANT__;
```

Comment:
Can remove the TODO

Reply:
I think I just need to re-word it since it still doesn't exist in the .new implementation. (Until @sebmarkbage tells me it's okay to add there as well.)